ssl/t1_trce.c

```diff
@@ -65,10 +65,6 @@ static const ssl_trace_tbl ssl_version_tbl[] = {
     {TLS1_1_VERSION, "TLS 1.1"},
     {TLS1_2_VERSION, "TLS 1.2"},
     {TLS1_3_VERSION, "TLS 1.3"},
-    /* TODO(TLS1.3): Remove these lines before release */
-    {TLS1_3_VERSION_DRAFT_26, TLS1_3_VERSION_DRAFT_TXT_26},
-    {TLS1_3_VERSION_DRAFT_27, TLS1_3_VERSION_DRAFT_TXT_27},
-    {TLS1_3_VERSION_DRAFT, TLS1_3_VERSION_DRAFT_TXT},
```

Comment:
(i.e., the draft versions might still be useful in tracing for a bit)

Reply:
I could go either way on this as it's only diagnostic stuff.  But then you get into "why not 28?" and "why 26" discussions.  Cleanest to just remove it all.

Reply:
For the same reasons as above I am strongly opposed to keeping these around. They should die die die!

Reply:
Exterminate. 
They should go.
We don't expect to see anything other than the RFC version going forward - correct?

Reply:
Correct (once the draft version implementations that are in the wild have withered away and died - which I expect to happen quickly)